packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -386,12 +386,22 @@ export interface CallWithChatAdapterSubscriptions {
  *
  * @public
  */
-export interface CallWithChatAdapter
+export interface CallWithChatAdapterCommon
   extends CallWithChatAdapterManagement,
     AdapterState<CallWithChatAdapterState>,
     Disposable,
     CallWithChatAdapterSubscriptions {}
 
+/**
+ * @public
+ */
+export interface CallWithChatAdapter extends CallWithChatAdapterCommon {
+  joinCall(microphoneOn?: boolean): Call | undefined;
+  startCall(participants: string[], options?: StartCallOptions): Call | undefined;
+  /* @conditional-compile-remove(PSTN-calls) */
+  startCall(participants: CommunicationIdentifier[], options?: StartCallOptions): Call | undefined;
```

Comment:
just something I'm noticing here but not part of this PR, I didn't think you can have two overloaded functions with the same number of arguments. Should this not need to be:
```suggestion
  startCall(participants: CommunicationIdentifier[] | string[], options?: StartCallOptions): Call | undefined;
```

Reply:
I mentioned this in another PR for startCall as well, I don't remember the detail of this one, but this was what we end up with in the end, @anjulgarg please correct me if I am wrong

Reply:
BTW, for "have two overloaded functions with the same number of arguments", we have some examples from "on" functions in calling sdk subscription. (But I still preferred "or" style than override if that is possible)